community/cloud-foundation/templates/backend_service/tests/integration/backend_service.bats

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bats
+
+source tests/helpers.bash
+
+TEST_NAME=$(basename "${BATS_TEST_FILENAME}" | cut -d '.' -f 1)
+
+# Create a random 10-char string and save it in a file.
+RANDOM_FILE="/tmp/${CLOUD_FOUNDATION_ORGANIZATION_ID}-${TEST_NAME}.txt"
+if [[ ! -e "${RANDOM_FILE}" ]]; then
+    RAND=$(head /dev/urandom | LC_ALL=C tr -dc a-z0-9 | head -c 10)
+    echo ${RAND} > "${RANDOM_FILE}"
+fi
+
+# Set variables based on the random string saved in the file.
+# envsubst requires all variables used in the example/config to be exported.
+if [[ -e "${RANDOM_FILE}" ]]; then
+    export RAND=$(cat "${RANDOM_FILE}")
+    DEPLOYMENT_NAME="${CLOUD_FOUNDATION_PROJECT_ID}-${TEST_NAME}-${RAND}"
+    # Replace underscores with dashes in the deployment name.
+    DEPLOYMENT_NAME=${DEPLOYMENT_NAME//_/-}
+    CONFIG=".${DEPLOYMENT_NAME}.yaml"
+    # test specific variables
+    export REGION="us-east1"
+    export RES_DESCRIPTION="This is the description"
+    export BC_DESCRIPTION="Backend description"
+    export TIMEOUT="35"
+    export ENABLE_CDN="false"
+    export SESSION="CLIENT_IP"
+    export REGIONAL_BALANCING_MODE="CONNECTION"
+    export GLOBAL_BALANCING_MODE="RATE"
+    export ZONE="us-central1-c"
```

Comment:
Are we using the ZONE value anywhere? Is the any other unused value?

Reply:
Sorry, I double checked and it's only used by the test YAML. If nothing interesting pops up during `enableCDN` troubleshooting, I'll remove the `ZONE`

Reply:
Should we do a check in an assert for the ZONE as well?

Reply:
Well, we can, but I probably wouldn't do that. The zone is used during test infrastructure setup only (zonal instance group), not in resources being tested. Backend service that is using that group, in fact, is global, and that is what we actually have an assertion for.

Reply:
Removed `ZONE` variable